PC02_Graffiti_EPPERSON.py

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: Max Epperson
+May 29, 2020
+'''
+
+from turtle import * #import the library of commands that you'd like to use
+
+colormode(255)
+
+# Create a panel to draw on. 
+panel = Screen()
+w = 750 # width of panel
+h = 750 # height of panel
+panel.setup(width=w, height=h) #600 x 600 is a decent size to work on. 
+#You can experiment by making it the size of your screen or super tiny!
+
+# Create a colorful background and add Bezos image to it
+image = "Bezos.gif"
+panel.bgcolor("lightsteelblue1")
+panel.bgpic(image)
+
+#=======Add your code here======
+home()
+showturtle()
+up()
+clear()
+goto(120,-90)
+clear()
+
+color(168,204,215) #piss jar base
+width(5)
+down()
+fillcolor(225,225,20)
+begin_fill()
+left(90)
+forward(50)
+right(45)
+goto(130,-30)
+right(45)
+forward(100)
+right(45)
+goto(240,-40)
+right(45)
+forward(150)
+right(45)
+goto(230, -200)
+right(45)
+forward(100)
+right(45)
+goto(120,-190)
+right(45)
+forward(100)
+end_fill()
+
+up()            #glass details
+goto(130, -190)
+down()
+right(90)
+forward(100)
+up()
+goto(130,-40)
+down()
+forward(100)
+
+up()            #jar lid base
+goto(130,-25)
+color("black")
+fillcolor("black")
+down()
+begin_fill()
+forward(100)
+left(90)
+forward(20)
+left(90)
+forward(100)
+left(90)
+forward(20)
+end_fill()
+ 
+up()           #jar lid lines
+goto(135,-6)
+color("white")
+width(2)
+down()
+forward(19)
+up()
+goto(145,-6)
+down()
+forward(19)
+up()
+goto(155,-6)
+down()
+forward(19)
+up()
+goto(165,-6)
+down()
+forward(19)
+up()
+goto(175,-6)
+down()
+forward(19)
+up()
+goto(185,-6)
+down()
+forward(19)
+up()
+goto(195,-6)
+down()
+forward(19)
+up()
+goto(205,-6)
+down()
+forward(19)
+up()
+goto(215,-6)
+down()
+forward(19)
+up()
+goto(225,-6)
+down()
+forward(19)
+
+up()           #piss bubbles
+goto(150,-70)
+color(199,193,22)
+down()
+dot(20)
+up()
+goto(170,-90)
+dot(10)
+goto(150,-100)
+dot(5)
+
+up()              #platter
+goto(50,-205)
+width(5)
+color("black")
+fillcolor(190,194,203)
+down()
+begin_fill()
+goto(310,-205)
+goto(240,-220)
+goto(120,-220)
+goto(50,-205)
+end_fill()
+
+
+up()                #speech box
+goto(65,50)
+fillcolor("white")
+down()
+begin_fill()
+goto(105,90)
+goto(325,90)
+goto(325,200)
+goto(105,200)
+goto(105,110)
+goto(65,50)
+end_fill()
+
+up()               #text in speech box
+goto(215,155)
+down()
+style =("Comic Sans",15,"italic")
+write("Thank god for my workers!", font=style , align="center")
+up()
+goto(215,135)
+write("They're always keeping me", font=style, align="center")
+up()
+goto(215,115)
+write("refreshed!", font=style, align="center")
+
```

Comment:
Add dot notation, soft code values
```suggestion
skin = (255,220,177)
handXcor = 120
handYcor = -220
#Draws hand that is holding up the piss platter 
handTurt.up()             
handTurt.goto(120,-220)
handTurt.width(3)
handTurt.color("black")
handTurt.fillcolor(skin)
handTurt.down()
handTurt.begin_fill()
handTurt.goto(handXcor+120,handYcor)
handTurt.goto(handXcor+190,handYcor-28)
handTurt.goto(handXcor+120,handYcor-28)
handTurt.goto(handXcor+10,handYcor-28)
handTurt.goto(handXcor, handYcor-18)
handTurt.goto(handXcor,handYcor)
handTurt.end_fill()
handTurt.goto(handXcor+60,handYcor)
handTurt.goto(handXcor+60,handYcor-10)
handTurt.goto(handXcor+90,handYcor-10)
handTurt.up()
handTurt.goto(handXcor,handYcor-10)
handTurt.down()
handTurt.goto(handXcor+50,handYcor-10)
handTurt.up()
handTurt.goto(handXcor,handYcor-9)
handTurt.down()
handTurt.goto(handXcor+50,handYcor-9)
```